dcpy/builds/load.py

```diff
@@ -16,49 +16,43 @@ def setup_build_environments(pg_client: postgres.PostgresClient):
     )
 
 
-def load_source_data(product: str):
-    product_path = REPO_ROOT_PATH / f"products/{product}"
-    build_name = build_metadata.build_name()
-
-    pg_client = postgres.PostgresClient(
-        schema=build_name,
-        database=f"db-{product}",
-    )
-
-    recipe_path = product_path / "recipe.yml"
-    recipe_lock_path = recipe_path.parent / "recipe.lock.yml"
+def load_source_data(recipe_path: Path):
+    recipe_lock_path = recipe_path.parent / f"{recipe_path}.lock.yml"
```

Comment:
Is this formatted string not going to be "recipe.yml.lock.yml"?

Reply:
true! will fix, I guess the template and pluto builds worked anyway because the `recipe_lock_path` is just passed along and used, and not expected to have a specific format

Reply:
formatted string looks fixed in this build log [here](https://github.com/NYCPlanning/data-engineering/actions/runs/6509328567/job/17680624154?pr=291#step:7:15)